### PR TITLE
improve how to add a new importer type doc section

### PR DESCRIPTION
**Pull request by lsmith77**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,7 +75,8 @@ admin overview panel using the event hook system, ie. `admin/tax-categories/`.
   - Implement the importer
  
     ```php
-    class FooImporter extends AbstractImporter
+    // alternatively extend from AbstractImporter
```
Comment (lsmith77) [inline README.md]: maybe we should also rename `AbstractImporter` to `AbstractPortPhpImporter` ?

Comment (solutionDrive-Alt) [reply README.md]: yes, or we get started with #7  and try to solve it without the AbstractImporter

Comment (lsmith77) [reply README.md]: which we also need for message queue support https://github.com/FriendsOfSylius/SyliusImportExportPlugin/issues/8

Comment (solutionDrive-Alt) [reply README.md]: Then I would say we should leave the name of the AbstractImporter be and focus on #7.

**Pull request merged**